compiler/noirc_evaluator/src/brillig/brillig_gen/constant_allocation.rs

```diff
@@ -23,7 +23,7 @@ pub(crate) enum InstructionLocation {
 }
 
 pub(crate) struct ConstantAllocation {
-    constant_usage: HashMap<ValueId, HashMap<BasicBlockId, Vec<InstructionLocation>>>,
+    constant_usage: HashMap<RawValueId, HashMap<BasicBlockId, Vec<InstructionLocation>>>,
```

Comment:
This one could also be `ResolvedValueId`.

Reply:
Done.